### PR TITLE
Add/stats package readme documentation

### PR DESCRIPTION
**Pull request by darssen**
#### Changes proposed in this Pull Request:

* Updated Readme for Stats Package

#### Jetpack product discussion

1203070774179820-as-1203101231018815/f

#### Does this pull request change what data or activity we track or use?

No
#### Testing instructions:

N/A

### CODE REVIEW

File: projects/packages/stats/README.md [hunk 1]
```diff
@@ -1,14 +1,96 @@
...
+
+This is done by enqueueing a JavaScript file in the footer of the site `https://stats.wp.com/e-202103.js` (the number changes every week but the file remains the same). 
+That JavaScript file, in turn, inserts a tracking pixel to the end of the page. For example: `https://pixel.wp.com/g.gif?lots_of_parameters_here`
+When that tracking pixel is loaded by the browser, it effectively sends information to WordPress.com about what needs to be logged on.
+
+### Parameters tracked
+
+#### Default parameters
+The default parameters being tracked are built with the `Automattic\Jetpack\Stats\TrackingPixel::build_view_data` function.
+- **v**:
```
Comment (darssen) [inline projects/packages/stats/README.md]: Not sure how to name this parameter

Comment (fgiannar) [reply projects/packages/stats/README.md]: (internal) Used to distinguish self-hosted sites from the ones hosted on WordPress.com. Enum: `ext, wpcom`

**Review by fgiannar — CHANGES_REQUESTED** [review]
Looks great, thanks for working on this! I believe we can :shipit: as soon as the conflicts are resolved :)

**Review by fgiannar — DISMISSED** [review]
LGTM!

File: projects/packages/stats/README.md (hunk 1 above)
Comment (darssen) [reply projects/packages/stats/README.md]: Thanks

**Review by fgiannar — DISMISSED** [review]
LGTM!

**Review by fgiannar — APPROVED** [review]
Re-approving

**Pull request merged**